specs/latest/2.0/index.html

```diff
@@ -785,27 +785,29 @@ <h2 class="no-toc">Table of contents</h2>
                             GLsizei height, GLint border, GLintptr offset);
   void compressedTexImage2D(GLenum target, GLint level, GLenum internalformat, GLsizei width,
                             GLsizei height, GLint border, ArrayBufferView srcData,
-                            optional GLuint srcOffset = 0);
+                            optional GLuint srcOffset = 0, optional GLuint srcLengthOverride = 0);
```

Comment:
Could you document the behavior of srcLengthOverride? i.e., what it defaults to if it's 0?